config/jobs/ppc64le-cloud/jenkins-infra/jenkins-infra-postsummit.yaml

```diff
@@ -0,0 +1,23 @@
+postsubmits:
+  ppc64le-cloud/jenkins-infra:
+    - name: load-jenkins-infra-job
+      branches:
+      - master
+      decorate: true
+      spec:
+        containers:
+          - image: quay.io/powercloud/jenkins-infra:0.1
+            envFrom:
+            - secretRef:
+                name: jenkins-secret
```

Comment:
@sudeeshjohn do we have this secret created?

Reply:
@mkumatag this secret is created in jenkins namespace in prow cluster. @sudeeshjohn will push the yaml file in secret repo

Reply:
These jobs run in the `test-pods` namespace, you might need to create there I guess.

Reply:
@mkumatag Created the secret in test-pods namespace

Reply:
cool

Reply:
@sudeeshjohn make sure you push changes to secrets project as well.